numba/__init__.py

```diff
@@ -56,6 +56,9 @@
 # Initialize hardware
 import numba.core.extending_hardware
 
+# Initialize typed containers
+import numba.typed
```

Comment:
@sklam Do you recall why this is here? Could it go into the CPU context?

Reply:
I think this is needed to register the typed containers for cases where they are used by implicitly; e.g. `d = {}`